Ashasindhu.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Asha Gadiraju",
+  "displayStatsPublic":true,
+  "type": "personal",
+  "bio": "Learning web development ,open source enthusiast",
+  "avatar":"https://github.com/Ashasindhu.png",
```

Comment:
hey @Ashasindhu, in new update we don't need avatar, so you can remove it. thanks for contributing